modules/AOW_WorkFlow/aow_utils.php

```diff
@@ -47,7 +47,8 @@ function getModuleFields(
     $view = 'EditView',
     $value = '',
     $valid = array(),
-    $override = array()
+    $override = array(),
+    $calledFromReports = false
```

Comment:
Hi @serhiisamko091184, this is not a good approach. Functions should not have params that are specific to the place they are called from. All parameters should be able to be used from anywhere.